code/game/objects/items/control_wand.dm

```diff
@@ -146,7 +186,7 @@
 	var/busy = FALSE
 
 /obj/item/door_remote/omni/access_tuner/afterattack(obj/machinery/door/airlock/D, mob/user)
```

Comment:
```suggestion
/obj/item/door_remote/omni/access_tuner/afterattack(obj/machinery/door/D, mob/user)
```
need to account for the fact that a non `/airlock` type will be provided as a param here regularly